frontend/src/lib/stores.js

```diff
@@ -0,0 +1,16 @@
+import words from '../languages/et/words.json'
+import { writable } from 'svelte/store'
+
+function randInt(min, max) {
+  return Math.floor(Math.random() * (max - min)) + min
+}
+
+function getRandomWord() {
+  return words[randInt(0, words.length - 1)]
+}
+
+export const currentWord = writable(getRandomWord())
+
+export function skipNext() {
+  currentWord.set(getRandomWord())
```

Comment:
Please change the logic so this checks if the new random word is the same as the previous one, and it keeps picking new words until it comes up with a new unique choice.

You will need `while` for that.